creator/settings/development.py

```diff
@@ -173,3 +173,5 @@
 AUTH0_AUD = 'https://kf-study-creator.kidsfirstdrc.org'
 CACHE_AUTH0_KEY = 'AUTH0_PUBLIC_KEY'
 CACHE_AUTH0_TIMEOUT = 86400
+
+CLIENT_ADMIN_SCOPE = 'download:file'
```

Comment:
I thought we wanted this to represent more of a role? Like `role:admin`. If not, then my next questions would be: should this be called `SERVICE_CLIENT_SCOPE` and should it be set to `'download:file upload:file'`

Reply:
We can set it to whatever we want to name it, `role:admin` may be more representative.

Reply:
Renamed